### DISCUSSION TITLE
Disable WebpackError: Unsafe builtin usage

### DISCUSSION THREAD

**Discussion by Floriferous** [gatsbyjs/gatsby#32672] [Help]
### Preliminary Checks

- [X] This issue is not a duplicate. Before opening a new issue, please search existing issues: https://github.com/gatsbyjs/gatsby/issues
- [X] This issue is not a question, feature request, RFC, or anything other than a bug report directly related to Gatsby. Please post those things in GitHub Discussions: https://github.com/gatsbyjs/gatsby/discussions

### Description

When I build, I have 3 npm packages that generate a `warn WebpackError: Unsafe builtin usage`:

- `https.get` in `simple-ddp`
- `fs.existsSync` in `postcss`
- `fs.statSync` in `browserslist`

These are not packages I'm planning to fork or change anything about, so I'd rather just be able to disable incremental builds, as we call `gatsby clean` before every build anyways.

Also, I can't confirm this, but I suspect this is breaking builds on circleCI, as the build step crashes without showing these warnings, and it fails right before where the warnings are supposed to show up.

### Reproduction Link

private

### Steps to Reproduce

1. Use above mentioned packages
2. run `gatsby build`

### Expected Result

I expect there to be no warning.

### Actual Result

There are multiple warnings.

### Environment

```shell
System:
    OS: macOS 11.4
    CPU: (16) x64 Intel(R) Core(TM) i9-9980HK CPU @ 2.40GHz
    Shell: 5.8 - /bin/zsh
  Binaries:
    Node: 12.14.0 - /usr/local/bin/node
    Yarn: 1.21.1 - /usr/local/bin/yarn
    npm: 6.13.4 - /usr/local/bin/npm
  Languages:
    Python: 2.7.16 - /usr/bin/python
  Browsers:
    Chrome: 92.0.4515.107
    Firefox: 82.0.2
    Safari: 14.1.1
  npmPackages:
    gatsby: ^3.10.2 => 3.10.2 
    gatsby-image: ^3.10.0 => 3.10.0 
    gatsby-plugin-canonical-urls: ^3.10.0 => 3.10.0 
    gatsby-plugin-csp: ^1.1.3 => 1.1.3 
    gatsby-plugin-google-tagmanager: ^3.10.0 => 3.10.0 
    gatsby-plugin-hotjar: ^1.1.1 => 1.1.1 
    gatsby-plugin-image: ^1.10.1 => 1.10.1 
    gatsby-plugin-manifest: ^3.10.0 => 3.10.0 
    gatsby-plugin-material-ui: ^3.0.1 => 3.0.1 
    gatsby-plugin-netlify: ^3.10.0 => 3.10.0 
    gatsby-plugin-no-sourcemaps: ^3.10.0 => 3.10.0 
    gatsby-plugin-preconnect: ^1.2.0 => 1.2.0 
    gatsby-plugin-prismic-previews: ^4.0.3 => 4.0.3 
    gatsby-plugin-react-helmet: ^4.10.0 => 4.10.0 
    gatsby-plugin-remove-serviceworker: ^1.0.0 => 1.0.0 
    gatsby-plugin-robots-txt: ^1.6.8 => 1.6.8 
    gatsby-plugin-sass: ^4.10.0 => 4.10.0 
    gatsby-plugin-sitemap: ^4.6.0 => 4.6.0 
    gatsby-source-filesystem: ^3.10.0 => 3.10.0 
    gatsby-source-prismic: ^4.0.2 => 4.0.2 
  npmGlobalPackages:
    gatsby-cli: 2.10.11
```

### Config Flags

None.

**Comment by LekoArts** [comment]
Hi!

> These are not packages I'm planning to fork or change anything about

You don't need to fork them, you'll need to adapt how you use them. Here's an example of how people used it: https://github.com/algolia/algoliasearch-client-javascript/issues/1259

> so I'd rather just be able to disable incremental builds

As the warning says already it'll disable the functionality by itself so you don't need to disable anything.

> as we call gatsby clean before every build anyways

Can you say why? It's nothing you should do really as you waste time & money with this.

> Also, I can't confirm this, but I suspect this is breaking builds on circleCI, as the build step crashes without showing these warnings, and it fails right before where the warnings are supposed to show up.

The code paths triggering these warnings only disable a certain branch/functionality, it won't affect the build breaking in my mind.
